constraints/cpu/BUILD

```diff
@@ -15,3 +15,12 @@ constraint_value(
     constraint_setting = "@platforms//cpu:cpu",
     visibility = ["//visibility:public"],
 )
+
+# TODO(cfrantz): Upstream to https://github.com/bazelbuild/platforms
+constraint_value(
+    # Based on the current riscv64 consrtaint in platforms, I wonder
+    # if this should be `riscv32` or `rv32imc`.
+    name = "rv32imc",
```

Comment:
I'm leaning towards keeping `rv32imc`. It looks like the generic `arm` target is slated for removal and there are a bunch of more specific 32-bit arm targets. `riscv64` is probably geared towards the non-embedded use case.

Reply:
Based on https://github.com/bazelbuild/platforms/pull/30, `riscv32` is the appropriate high-level cpu descriptor.